### DISCUSSION TITLE
Vr - Quest 2

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#407] [! Welcome - readme first!]
(i'm only using the right controller) but I can only seem to get access to the triggers, don't seem to get a read on the other buttons.

**Comment by pandrr** [comment]
worked when developed op.

please provice example to reproduce

**Reply by marioepsley** [reply]
shared a patch with you :)

**Comment by pandrr** [comment]
thanks for patch. works on dev.cables.gl now

**Reply by archgary** [reply]
Any updates on this, would like to add some interaction with my VR project. 
The vrController OP doesn't get any data/info from my Quest 2, like the joystick, triggers, and buttons. 
Thank you so much for your answers in advance.

**Reply by pandrr** [reply]
you can use the updated op on the in-development version of cables, which you can reach at https://dev.cables.gl

**Reply by archgary** [reply]
Hi Pandrr,

Thanks for the information. And the buttons work in the in-dev version.

Still have 2 questions here:
1. I don't seem to get the Axis working, is it only my issue? (what is the value range of the Axis?)
2. If I use the in-dev version for my work, does it mean that if I want to share my work, I will have to share them with this in-dev version instead. Or how would this work?

Thank you for your answer. Appreciate all your hard work on this amazing tool. 
Looking forward to your reply.

Best,
archgary
